libraries/standard/mqtt/src/mqtt.c

```diff
@@ -291,15 +355,24 @@ MQTTStatus_t MQTT_Disconnect( MQTTContext_t * const pContext )
         bytesSent = sendPacket( pContext,
                                 pContext->networkBuffer.pBuffer,
                                 packetSize );
+        IotLogDebugWithArgs( "Sent %d bytes of disconnect packet. ",
```

Comment:
This log seems duplicated in the else part.

Reply:
Thanks. Fixed it